testapps/iOSImageFormatProfilingTestApp/test/ViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+import UIKit
+
+class ResettableView: UIView {
+	override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+		super.touchesEnded(touches, with: event)
+		subviews.forEach { $0.removeFromSuperview() }
+		setNeedsLayout()
+		setNeedsDisplay()
+		layoutIfNeeded()
```

Comment:
Can we add comments explaining why we do things this way? like calling all of setNeedsLayout/setNeedsDisplay/layoutIfNeeded is not a common pattern according to all other guidelines